cpp-client/tests/clean_shutdown_test.cc

```diff
@@ -0,0 +1,29 @@
+#include <chrono>
```

Comment:
Since this is not finished I would say for now let's just remove?  No worse than before in that we don't have test cases for this.  I promise I'll add a little cpp-examples program that creates and destroys clients in a loop in a later PR.

Reply:
ok deleted :-)